diff --git a/SimpleChatter.cpp b/SimpleChatter.cpp
index 9ec0f51..678c69f 100644
--- a/SimpleChatter.cpp
+++ b/SimpleChatter.cpp
@@ -5,7 +5,7 @@ using namespace std;
 
 int conversation() {
     int conv;
-    conv = 1 + rand() % 15;
+    conv = 1 + rand() % 20;
     switch (conv)
     {
     case 1: { cout << "Хех, мда.\n"; break; }
@@ -23,21 +23,26 @@ int conversation() {
     case 13: { cout << "Как-то так, да.\n"; break; }
     case 14: { cout << "Мхм.\n"; break; }
     case 15: { cout << "Ага, конечно.\n"; break; }
+    case 16: { cout << "Oh yeah?\n"; break; }
+    case 17: { cout << "Мне очень интересно.\n"; break; }
+    case 18: { cout << "Тоже мне.\n"; break; }
+    case 19: { cout << "Ага, да.\n"; break; }
+    case 20: { cout << "Воу. Я правда удивлен. Типо.\n"; break; }
     }
     return conv;
 }
 
 int main()
 {
-    system("chcp 1251");
-    system("cls");
+    setlocale(LC_ALL, "rus");
     int lenghth, reply;
     char conver[255];
     cout << R"~(Так о чём ты там поболтать хотел? Я собеседник довольно интересный, разговор поддержать могу. Только покороче пиши, ок?)~"<<endl;
-TALK:
-    cout << ">";
-    cin.getline(conver,128);
-    lenghth = strlen(conver);
+    while (true) {
+        cout << ">";
+        cin.getline(conver, 128);
+        lenghth = strlen(conver);
         random_device rand;
-    reply = conversation(); goto TALK;
+        reply = conversation();
+    }
 }
